test/e2e/lazy_activation_policy_test.go

```diff
@@ -90,6 +103,92 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
+				// 2. update policy placement with clusterAffinities
+				changePlacementTargetCluster(policy, modifiedCluster)
+				// 3. the propagation of deployment changed
+				framework.WaitDeploymentDisappearOnCluster(originalCluster, namespace, deploymentName)
+				waitDeploymentPresentOnCluster(modifiedCluster, namespace, deploymentName)
+			})
+		})
+
+		ginkgo.Context("Immediate to lazy", func() {
+			ginkgo.BeforeEach(func() {
+				// remove lazy activationPreference field
+				policy.Spec.ActivationPreference = ""
+			})
+
+			// Simple Case 4 (Immediate to lazy)
+			// refer: https://github.com/karmada-io/karmada/blob/master/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#simple-case-4-immediate-to-lazy
+			ginkgo.It("Simple Case 4 (Immediate to lazy)", func() {
+				ginkgo.By("step 1: deployment propagate success", func() {
+					waitDeploymentPresentOnCluster(originalCluster, namespace, deploymentName)
+				})
+
+				ginkgo.By("step 2: after policy updated (cluster=member2, activationPreference=lazy), deployment's propagation status unchanged", func() {
+					// 1. activationPreference=lazy
+					policy.Spec.ActivationPreference = policyv1alpha1.LazyActivation
+					// 2. update policy placement with clusterAffinities
+					changePlacementTargetCluster(policy, modifiedCluster)
+					// 3. wait to distinguish whether the state will not change or have no time to change
+					time.Sleep(pollInterval)
+					waitDeploymentPresentOnCluster(originalCluster, namespace, deploymentName)
+					framework.WaitDeploymentDisappearOnCluster(modifiedCluster, namespace, deploymentName)
+				})
+
+				ginkgo.By("step3: resource would propagate when itself updated", func() {
+					// 1. update annotation of the deployment
+					updateDeploymentManually(deployment)
+					// 2. the propagation of deployment changed
+					framework.WaitDeploymentDisappearOnCluster(originalCluster, namespace, deploymentName)
+					waitDeploymentPresentOnCluster(modifiedCluster, namespace, deploymentName)
+				})
+			})
+		})
+
+		// Combined Case 4 (Policy preemption)
+		// refer: https://github.com/chaosi-zju/karmada/blob/lazy-proposal/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#combined-case-4-policy-preemption
+		ginkgo.It("Policy preemption", func() {
+			ginkgo.By("step 1: deployment propagate success when policy created before it", func() {
+				waitDeploymentPresentOnCluster(originalCluster, namespace, deploymentName)
+			})
+
+			ginkgo.By("step 2: create PP2 (match nginx, cluster=member2, lazy, priority=2, preemption=true)", func() {
+				framework.CreatePropagationPolicy(karmadaClient, policyHigherPriority)
+				// 1. label of policy name changed
+				framework.WaitDeploymentFitWith(kubeClient, namespace, deploymentName, func(deployment *appsv1.Deployment) bool {
+					policyNameLabel := util.GetLabelValue(deployment.GetLabels(), policyv1alpha1.PropagationPolicyNameLabel)
```

Comment:
We will remove the label of the policy in a moment. You can use annotation instead.